### PR TITLE
Improve readability of messages

### PR DESCRIPTION
**Pull request by rmhartog**
The code now uses the `format` method instead of String concatenation.

Please review!

### CODE REVIEW

**Comment by VHellendoorn** [comment]
Please cite a source that proves this is more readable. Although we have not formally adopted a convention on this, the use of `printf` to me suggests an over-reliance on habits from previous programming languages (*cough* C). I personally never saw `println` in Java as a problem, _provided_ there are not too many variables involved.

File: src/runner/Runner.java [hunk 1]
```diff
@@ -35,20 +35,20 @@ private static void printSelection(int count, Map<String, Integer> counts, List<
...
-				System.out.println("Your place of dining today is: " + restaurant + " (" + perc + "%)");
+				System.out.format("Your place of dining today is: %s (%d%%)%n", restaurant, perc);
 				break;
 			}
 		}
 		System.out.println("The options ranked:");
 		for (String restaurant : restaurantsOrdered) {
 			perc = (int) (100 * ((double) counts.get(restaurant) / count));
-			System.out.println(restaurant + "\t" + perc + "%" + (restaurant.matches(EXCLUDED) ? "*" : ""));
+			System.out.format("%s\t%d%%%s%n", restaurant, perc, restaurant.matches(EXCLUDED) ? "*" : "");
```
Comment (VHellendoorn) [inline src/runner/Runner.java]: Here, I agree that the print should be changed, but what bothers me actually is the in-line conditional statement. Extracting this to a variable, regardless of whether we use `println` or `printf` would improve this sufficiently IMO.

File: src/runner/Runner.java [hunk 2]
```diff
@@ -35,20 +35,20 @@ private static void printSelection(int count, Map<String, Integer> counts, List<
...
 		}
-		System.out.println("*Disregarded, as we hate these people.");
+		System.out.println("* Disregarded, as we hate these people.");
```
Comment (VHellendoorn) [inline src/runner/Runner.java]: I suppose I can live with this.

**Comment by VHellendoorn** [comment]
My apologies, I used `printf` in several comments where I clearly meant `out.format` (effectively the same I assume). Please read as if I said the latter.

**Comment by rmhartog** [comment]
@anandsaw and @koudshoorn please weigh in on this PR.

**Comment by anandsaw** [comment]
Yeah, we are really not going to use format printing, I am sorry put all this effort in, it was much appreciated, however I feel that in Java using printf is quite useless.

**Comment by anandsaw** [comment]
This is going to die.

**Comment by VHellendoorn** [comment]
Please commit the space in the last line instead of this PR

**Comment by rmhartog** [comment]
Please review this post http://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java

**Comment by anandsaw** [comment]
There are performance issue unfortunately:
The timing results are as follows:

Concatenation = 265 millisecond
Format = 4141 millisecond

This branch is going to die.

**Pull request closed without merging**